torch/_tensor_docs.py

```diff
@@ -1834,6 +1859,17 @@ def add_docstr_all(method, docstr):
 See :func:`torch.isreal`
 """)
 
+add_docstr_all('is_coalesced',
+               r"""
+is_coalesce() -> bool
+
+Returns True if :attr:`self` tensor is a coalesced sparse COO tensor
+which has unique indices in lexicographical ordering.  If :attr:`self`
+is not a sparse COO tensor, this throws an error.
```

Comment:
It's worth noting that coalesceness is a boolean property maintained on the tensor, separate from the actual state of the data. That is to say, if you *accidentally* end up with a coalesced tensor where PyTorch cannot prove that the result would always be coalesced, `is_coalesced` may still return false. In general, coalesce status is data independent.

Reply:
Yes, I agree. I propose the following fix:
```
Returns True if :attr:`self` tensor is a sparse COO tensor that has been coalesced.
```

Reply:
Fixed.